src/components/v2/V2Project/V2PayButton/V2ConfirmPayModal/index.tsx

```diff
@@ -203,6 +243,24 @@ export function V2ConfirmPayModal({
           >
             {formatWad(ownerTickets, { precision: 0 })}
           </Descriptions.Item>
+          {nftReward ? (
+            <Descriptions.Item
+              label={
+                <TooltipLabel
+                  label={t`NFT rewards`}
+                  tip={
+                    <Trans>
+                      Supporters receive this NFT for contributing at least{' '}
```

Comment:
I think this copy should be directed at the user who is paying. wdyt about

```suggestion
                      You receive an NFT for contributing{' '}
```